wgsl/index.bs

```diff
@@ -1459,45 +1461,53 @@ following table:
   <tr><td>[[[=stride=](|Q|)]]<br> array<|E|>
       <td>[=AlignOf=](|E|)
       <td>N<sub>runtime</sub> * |Q|
-  <tr><td>atomic<|T|>
-      <td>[=AlignOf=](|T|)
-      <td>[=SizeOf=](|T|)
 </table>
 
 
 #### Structure Layout Rules ####  {#structure-layout-rules}
 
-Each structure member has a default size and alignment value. These values are
-used to calculate each member's byte offset from the start of the structure.
+Each structure |S| member M<sub>N</sub> has a size and alignment value, denoted
+by [=SizeOfMember=](|S|, M<sub>N</sub>) and [=AlignOfMember=](|S|, M<sub>N</sub>), respectively.
+The member sizes and alignments are used to calculate each member's byte offset from the start of the structure.
+
+Structures member size and alignment values default to the member type `T`'s
+[=SizeOf=](T) and [=AlignOf=](T) values.
 
-Structure members will use their type's size and alignment, unless the
-structure member is explicitly annotated with [=attribute/size=] and / or
-[=attribute/align=].  decorations, in which case those member decorations take
-precedence.
+If a structure member is decorated with the [=attribute/size=] decoration, then the structure
+member will use the value of the decoration for its size instead of its type's size.
+
+If a structure member is decorated with the [=attribute/align=] decoration, then the structure
+member will use the value of the decoration for its alignment instead of its type's alignment.
 
 The first structure member always has a zero byte offset from the start of the
 structure.
 
 Subsequent members have the following byte offset from the start of the structure:
 <p algorithm="structure member offset">
-  [=OffsetOf=](|S|, M<sub>N</sub>) = [=roundUp=]([=AlignOf=](|S|, M<sub>N</sub>), [=OffsetOf=](|S|, M<sub>N-1</sub>) + [=SizeOf=](|S|, M<sub>N-1</sub>)<br>
+  [=OffsetOfMember=](|S|, M<sub>N</sub>) = [=roundUp=]([=AlignOfMember=](|S|, M<sub>N</sub>), [=OffsetOfMember=](|S|, M<sub>N-1</sub>) + [=SizeOfMember=](|S|, M<sub>N-1</sub>)<br>
   Where M<sub>N</sub> is the current member and M<sub>N-1</sub> is the previous member
 </p>
 
 Structure members must not overlap. If a structure member is decorated with the
 [=attribute/size=] attribute, the value must be at least as large as the
-default size of the member's type.
+size of the member's type:
+
+<p algorithm="member size constraint">
+  [=SizeOfMember=](|S|, M<sub>N</sub>) >= [=SizeOf=](T)<br>
```

Comment:
nit: this   `>=` should probably be `&ge;` so it renders as &ge;

Reply:
Done